docs/modules/release-notes/pages/all-releases/4_13_2S.adoc

```diff
@@ -0,0 +1,115 @@
+= Kobiton 4.13.2S release notes
+:navtitle: Kobiton 4.13.2S release notes
+
+== New OS support
+
+Kobiton 4.13.2S adds support for the following:
+
+* iOS/iPadOS 17.4.1, 17.5 Beta, 17.5.1
+* Mac OS Sonoma 14.4.1, 14.5
+* Android 15 Beta 3
+
+[NOTE]
+====
+* At this time, we recommend hosting iOS 17+ devices via Standard mode deployments for improved performance.
+* For Android 15 Beta 3, the cleanup policy cannot remove a signed-in account.
+* For iOS/iPadOS 18 Beta, network payload capture is not supported.

```

Comment:
@Hikkigaya Remove this line as iOS 18 Beta is not yet supported.

Reply:
Changes made